bootstrap.d/dev-libs.yml

```diff
@@ -767,3 +767,24 @@ packages:
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
         quiet: true
+
+  - name: json-c
+    source:
+      subdir: ports
+      git: 'https://github.com/json-c/json-c'
+      branch: 'master'
```

Comment:
We should build from a tag, or if that is not possible, a fixed commit to avoid new commits suddenly breaking our build.
Building from a fixed point also allows for adding version information.